api/src/main/java/com/gentics/mesh/etc/config/MeshOptions.java

```diff
@@ -211,6 +215,17 @@ public MeshOptions setStorageOptions(GraphStorageOptions storageOptions) {
 		return this;
 	}
 
+	@JsonProperty("s3binary")
+	public S3Options getS3Options() {
```

Comment:
Even though the `S3Options` do not have a validation method right now, I would still add them in the `MeshOptions` validate() method.